cmd/bundle/main.go

```diff
@@ -73,6 +73,13 @@ func Do(ctx context.Context) error {
 		return fmt.Errorf("mandatory flag --image is not set")
 	}
 
+	// sanity-check the endpoint, if hostname extraction fails, ignore that failure
```

Comment:
Nit: Trying to avoid ableist language.
```suggestion
	// check the endpoint, if hostname extraction fails, ignore that failure
```